libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,400 @@
+              "@id": "rockfish",
+              "CandidateIds": ["rockfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "triggerfish",
+              "CandidateIds": ["triggerfish"]
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "write-in-0",
+              "isWriteIn": true
+            },
+            {
+              "@type": "CVR.CandidateSelection",
+              "@id": "write-in-1",
+              "isWriteIn": true
+            }
+          ],
+          "PrimaryPartyId": "1"
+        },
+        {
+          "@type": "CVR.BallotMeasureContest",
+          "@id": "fishing",
+          "Name": "Ballot Measure 3",
+          "ContestSelection": [
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "yes",
+              "Selection": "yes"
+            },
+            {
+              "@type": "CVR.BallotMeasureSelection",
+              "@id": "no",
+              "Selection": "no"
+            }
+          ],
+          "PrimaryPartyId": "1"
+        }
+      ],
+      "ElectionScopeId": ["sample-county"]
+    }
+  ],
+  "GpUnit": [
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "sample-county",
+      "Name": "Sample County",
+      "Type": "other"
+    },
+    {
+      "@type": "CVR.GpUnit",
+      "@id": "precinct-1",
+      "Name": "Precinct 1",
+      "Type": "precinct"
+    }
+  ],
+  "Batch": [
+    {
+      "@type": "VX.Batch",
+      "@id": "batch-2",
+      "BatchLabel": "Batch 2"
+    }
+  ],
+  "CVR": [
+    {
+      "@type": "CVR.CVR",
+      "BallotStyleId": "2F",
+      "BallotStyleUnitId": "precinct-1",
+      "PartyIds": ["1"],
+      "CreatingDeviceId": "scanner-1",
+      "BatchId": "batch-2",
+      "ElectionId": "some-id",
+      "UniqueId": "id-168",
+      "vxBallotType": "absentee",
+      "vxTestBallot": true,
+      "CurrentSnapshotId": "id-168-original",
+      "CVRSnapshot": [
+        {
+          "@id": "id-168-original",
+          "@type": "CVR.CVRSnapshot",
+          "Type": "original",
+          "CVRContest": [
+            {
+              "@type": "CVR.CVRContest",
+              "ContestId": "best-animal-fish",
+              "CVRContestSelection": [],
+              "Undervotes": 1
+            },
+            {
+              "@type": "CVR.CVRContest",
+              "ContestId": "aquarium-council-fish",
+              "CVRContestSelection": [],
+              "Undervotes": 2
+            },
+            {
+              "@type": "CVR.CVRContest",
+              "ContestId": "fishing",
+              "CVRContestSelection": [
+                {
+                  "@type": "CVR.CVRContestSelection",
+                  "ContestSelectionId": "yes",
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "IsAllocable": "yes",
```

Comment:
I meant to take this out but, for demo purposes, there's this `IsAllocable` field which can be used to indicate whether a mark is supposed to be counted based on contest rules. We could include this on all marks, or just where it's "no", but it's not required so I'll just leave it out.